pkg/serviceability/logrus.go

```diff
@@ -0,0 +1,36 @@
+package serviceability
+
+import (
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+// InitLogrus initializes logrus by setting a loglevel for it.
+func InitLogrus(level string) {
```

Comment:
Who else uses this?   I don't see any uses in openshift/origin.  I remember a usage in image-registry I think, but I don't recall a second one.

Reply:
the library that the imagesignature feature relies on, uses logrus.  those are the logs i'm trying to get enabled w/ this.